CIP-0098/README.md

```diff
@@ -0,0 +1,474 @@
+---
+CIP: 98?
```

Comment:
```suggestion
CIP: ?
```

Reply:
Editors will assign a number in a CIP Editors meeting 🤓

Reply:
I understand 🙂️